scalatest/src/main/scala/org/scalatest/tools/DashboardReporter.scala

```diff
@@ -112,7 +112,7 @@ private[scalatest] class DashboardReporter(directory: String,
   //
   // Throws exception for specified unexpected event.
   //
-  def unexpectedEvent(e: Event): Unit = {
+  def unexpectedEvent(e: Event | Null): Unit = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.